var/spack/repos/builtin/packages/py-fortran-language-server/package.py

```diff
@@ -15,7 +15,7 @@ class PyFortranLanguageServer(PythonPackage):
 
     maintainers = ['AndrewGaspar']
 
-    version('1.11.1', sha256='8f03782dd992d6652a3f2d349115fdad3aa3464fee3fafbbc4f8ecf780166e3c')
+    version('1.12.0', sha256='5cda6341b1d2365cce3d80ba40043346c5dcbd0b35f636bfa57cb34df789ff17')
```

Comment:
Can you keep the old versions too? Also, can you update the URL so that at least one of these versions is present in the URL? It isn't needed for most things, it's only taken into account with `spack url summary`, a debugging tool.